Tests/test_image_getdominantcolors.py

```diff
@@ -0,0 +1,22 @@
+from .helper import hopper
+
+
+def test_getdominantcolors():
+    def getdominantcolors(mode, numcolors=None):
+        im = hopper(mode)
+
+        if numcolors:
+            colors = im.getdominantcolors(numcolors)
+        else:
+            colors = im.getdominantcolors()
```

Comment:
```suggestion
    def getdominantcolors(mode):
        im = hopper(mode)
        colors = im.getdominantcolors()
```
Just removing unused code, since this function is never called with `numcolors`.